examples/yaml-highlight/yaml_hl.py

```diff
@@ -37,14 +37,14 @@ def __setstate__(self, state):
 class YAMLHighlight:
 
     def __init__(self, options):
-        config = yaml.load(file(options.config, 'rb').read())
+        config = yaml.load(open(options.config, 'rb').read())
```

Comment:
Better, but it leaves an open file descriptor hanging... you should either close it or (much better) use a context manager:
```suggestion
        with open(options.config, 'rb').read() as yaml_file:
            config = yaml.load(yaml_file.read())
```